src/blocks/remote-data-container/hooks/useRemoteData.ts

```diff
@@ -94,7 +94,6 @@ interface UseRemoteDataInput {
 // don't need an intermediate state update / re-render.
 export function useRemoteData( {
 	blockName,
-	enabledOverrides = [],
```

Comment:
Remove from input type above, please

Reply:
Fixed in https://github.com/Automattic/remote-data-blocks/pull/383/commits/8acf0b23990b72c39477fec5fe8294f5fe3992c1